src/index.ml

```diff
@@ -324,6 +324,26 @@ module Make (K : Key) (V : Value) (IO : IO) = struct
         iter_io (fun e -> Tbl.replace mem e.key e.value) io;
         Some { io; mem }
     in
+    let log_async_path = log_async_path root in
+    if (not readonly) && (not fresh) && Sys.file_exists log_async_path then
+      may
+        (fun log ->
+          let io =
+            IO.v ~fresh ~readonly ~generation:0L ~fan_size:0L log_async_path
```

Comment:
Can you explicitly pass `~fresh:false ~readonly:false` here? I'm not sure _why_ I want it, but I do :slightly_smiling_face:

Reply:
I put it for `readonly`, I have a different handling of `fresh` in the last commit to still clean the file when `fresh` is `true`